packages/create-llama/templates/types/simple/fastapi/app/engine/index.py

```diff
@@ -36,3 +38,7 @@ def get_index():
         index = load_index_from_storage(storage_context,service_context=service_context)
         logger.info(f"Finished loading index from {STORAGE_DIR}")
     return index
+
+def get_chat_engine():
+    index = get_index()
+    return index.as_chat_engine()
```

Comment:
just remove `get_index` and use simple chat (like in nextjs/express):

```suggestion
    return   SimpleChatEngine.from_defaults(service_context=create_base_context())
```

please test, I didn't test this snippet